tests/ha/test_control_pod_restart.py

```diff
@@ -383,3 +401,286 @@ def test_failover_control_pod_kubectl(self):
+        thread_del.join()
+        LOGGER.info("Step 8.1: Verify status for In-flight DELETEs during control pod "
+                    "failover to %s", failover_node)
+        del_resp = tuple()
+        while len(del_resp) != 2:
+            del_resp = del_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        if not del_resp:
+            assert_utils.assert_true(False, "Background process failed to do deletes")
+        fail_del_bkt = del_resp[1]
+        rem_bkts_aftr_del = s3_test_obj.bucket_list()[1]
+        assert_utils.assert_false(len(fail_del_bkt),
+                                  f"Bucket deletion failed when cluster was online {fail_del_bkt}")
+        assert_utils.assert_equal(len(rem_bkts_aftr_del), remaining_bkt,
+                                  f"{del_bucket} buckets should get deleted, only "
+                                  f"{wr_bucket - len(rem_bkts_aftr_del)} were deleted")
+        LOGGER.info("Step 8.1: Verified status for In-flight DELETEs during control pod "
+                    "failover to %s", failover_node)
+
+        LOGGER.info("Step 8.2: Verify status for In-flight WRITEs during control pod "
+                    "failover to %s", failover_node)
+        wr_resp = dict()
+        while len(wr_resp) != 2:
+            wr_resp = wr_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        pass_logs = list(x[1] for x in wr_resp["pass_res"])
+        fail_logs = list(x[1] for x in wr_resp["fail_res"])
+        resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs)
+        assert_utils.assert_false(len(resp[1]), f"Logs which contain failures: {resp[1]}")
+        resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs)
+        assert_utils.assert_false(len(resp[1]), f"Logs which contain failures: {resp[1]}")
+        LOGGER.info("Step 8.2: Verified status for In-flight WRITEs during control pod %s "
+                    "failover", failover_node)
+
+        LOGGER.info("Step 8.3: Verify status for In-flight READs/Verify DI during control pod "
+                    "failover to %s", failover_node)
+        rd_resp = dict()
+        while len(rd_resp) != 2:
+            rd_resp = output_rd.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        pass_logs = list(x[1] for x in rd_resp["pass_res"])
+        fail_logs = list(x[1] for x in rd_resp["fail_res"])
+        resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs)
+        assert_utils.assert_false(len(resp[1]), f"Logs which contain failures: {resp[1]}")
+        resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs)
+        assert_utils.assert_false(len(resp[1]), f"Logs which contain failures: {resp[1]}")
+        LOGGER.info("Step 8.3: Verified status for In-flight READs/Verify DI during control pod"
+                    " failover to %s", failover_node)
+        LOGGER.info("Step 8: Verified status for In-flight READs/WRITEs/DELETEs during control "
+                    "pod failover to %s", failover_node)
```

Comment:
in all above checks why can't we check all pass and fail logs in one single logs as all are expected to pass?